### PR TITLE
chore: update Windows VHD to include 8B patches

### PR DESCRIPTION
**Pull request by mainred**
**Reason for Change**:

August 11 patches for windows are released

https://support.microsoft.com/en-us/help/4565349

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -190,9 +190,9 @@ function Install-WindowsPatches
     # Windows Server 2019 update history can be found at https://support.microsoft.com/en-us/help/4464619
     # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 
-    # KB4558998 contains July 2020 cumulative updates for Windows Server 2019
-    # https://www.catalog.update.microsoft.com/Search.aspx?q=4558998
-    $patchUrls = @("http://download.windowsupdate.com/c/msdownload/update/software/secu/2020/07/windows10.0-kb4558998-x64_6da68fe659dacb747458ab3a431c3546ce7765b5.msu")
+    # KB4558998 contains Auguest 11, 2020 cumulative updates for Windows Server 2019
```
Comment (jsturtevant) [inline vhd/packer/configure-windows-vhd.ps1:193]:
```suggestion
    # KB4558998 contains August 11, 2020 cumulative updates for Windows Server 2019
```

**Comment by jsturtevant** [comment]
The base image the VHD is based on, also needs to be updated:

https://github.com/Azure/aks-engine/blob/c273c5163962e9ef42e01c4547de93b635361084/vhd/packer/windows-vhd-builder.json#L29

should be updated to `17763.1339.2007101755`

```
z vm image show --urn MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-with-Containers-smalldisk:latest
{
  "automaticOsUpgradeProperties": {
    "automaticOsUpgradeSupported": false
  },
  "dataDiskImages": [],
  "hyperVgeneration": "V1",
  "id": "/Subscriptions/b9d9436a-0c07-4fe8-b779-2c1030bd7997/Providers/Microsoft.Compute/Locations/westus/Publishers/MicrosoftWindowsServer/ArtifactTypes/VMImage/Offers/WindowsServer/Skus/2019-Datacenter-Core-with-Containers-smalldisk/Versions/17763.1339.2007101755",
  "location": "westus",
  "name": "17763.1339.2007101755",
  "osDiskImage": {
    "operatingSystem": "Windows",
    "sizeInBytes": 32214352384,
    "sizeInGb": 31
  },
  "plan": null,
  "tags": null
}
```

**Comment by mainred** [comment]
@jsturtevant . Thanks for your comment.
To understand the base image change, I think it's because a windows patch contains changes based on the last month's base image,  cc @AbelHu

**Pull request merged**